bciers/apps/registration/app/components/operations/registration/FacilityInformationForm.tsx

```diff
@@ -68,8 +68,8 @@ const FacilityInformationForm = ({
   const [formState, setFormState] = useState(formData ?? {});
   const [isSubmitting, setIsSubmitting] = useState(false);
   // Get the list of sections in the LFO schema - used to unnest the formData
-  const formSectionListLFo = Object.keys(
-    facilitiesLfoSchema.properties as RJSFSchema,
+  const formSectionListSfo = Object.keys(
```

Comment:
I don't know why we were using the LFO one--this variable is used for SFOs later. Previously SFO and LFO had the same sections, so didn't matter